packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -209,7 +209,7 @@ export class MultiSelectExample extends React.PureComponent<IExampleProps, IMult
         this.selectFilms([film]);
     }
 
-    private selectFilms(filmsToSelect: IFilm[]) {
+    private selectFilms(filmsToSelect: readonly IFilm[]) {
```

Comment:
this seems like a breaking change for consumers, no? looks like you were getting TS compilation errors until you changed this.

Reply:
Yeah, that was what I was talking about here: https://github.com/palantir/blueprint/pull/5171#discussion_r828450664 

It is a compile time break, but if people were actually modifying the array I would imagine the select wouldn't be working.

However, it's easy enough to make a mutable copy of the item list and pass that to the callback

Reply:
yeah, i'd prefer that to avoid upgrade surprises right now. I don't want to force consumers to add the `readonly ` even if it's easy to do so. we can make the compile break in v5.0 (v4 is already closed for breaking changes)